### ISSUE TITLE
VS - Adding Durable Function via wizard rollsback DurableTask assembly version

### ISSUE THREAD

**Issue by andreujuanc** [Azure/azure-functions-durable-extension#211]
Hello,

Everythings works fine, but when I tried to add a new function via

**Scenario:** 
VS Version: 15.6.4
Win10 x64
Azure function project with durable extension added: 1.2.0-beta3

**Steps to reproduce:**

`Project context menu > Add> Azure Function> Add > Select Durable AF`

**Expected**
Just the Function.x.cs should be added.

**What actually happens**
The class file is added but changes the version of the durableTasks assembly to a previous one, and my build breaks.

//TODO: INSERT GIF HERE

Let me know if it's reproducible.

Regards

**Comment by cgillum** [comment]
Whoa, interesting. @ahmelsayed what is the right way to report VS template bugs for functions?

**Comment by ahmelsayed** [comment]
/cc @soninaren and @bzhu94 
Naren it looks like the version of the nuget package is [defined in the template repo](https://github.com/Azure/azure-functions-templates/blob/7f1b6ebaff7598551b3be92a8562a30f342ea469/Functions.Templates/Templates/DurableFunctionsOrchestration-CSharp/build.config/template.json#L37) It seems that v2.x HEAD has `1.2.0-beta3` but maybe VS is running an older version. 

Is there a way to tell VS to not downgrade a dependency? probably only upgrades make sense.

**Comment by soninaren** [comment]
Yeah, this has been a bug with VS. We don't really have a way to tell VS to not downgrade. That is why removed some explicit nuget dependency from templates to extension versions.  This should be resolved when vsix update for v2 is out.

@mlorbetske is there a workaround/fix for the bug.

**Comment by mlorbetske** [comment]
This is the first we're hearing of it, so there's no workaround that I'm aware of. @phenning - any ideas if there'd be a way to detect whether the package is already present & at a higher version in the install NuGet package post-action & skip the install if it is?

**Comment by andreujuanc** [comment]
@mlorbetske IMHO, best approach would be just skip it if package is present. Better not to mess with versions and stuff. Can cause builds to break and environment inconsistencies.

**Comment by phenning** [comment]
Sure, we can skip the install in that case, we just need to query IVsPackageInstallerServices for the list of installed packages. We have prior art around doing this for updating the version of CodeDom packages in the templates.

**Comment by soninaren** [comment]
Have added it here.
https://github.com/Azure/Azure-Functions/issues/753
